nucypher_cli/.nucypher.ini

```diff
@@ -0,0 +1,49 @@
+; ==================================
+;     ~ MACHINE TEMPLATE ONLY ~
+;
+;   DO NOT USE THIS CONFURATION FILE.
+;
+;   It is used to initialize default
+;   configuration files; However,
+;   This file may be useful as a
+;   form of documentation when
+;   configuring your own node.
+;
+; ==================================
+;
+;[nucypher]
+;mode=decentralized
+;simulation=True
+;
+;[character]
+;start_learning_on_same_thread=True
+;always_be_learning=True
+;abort_on_learning_error=False
+;
+;[network.cryptography]
+;tls_curve=secp256r1
```

Comment:
Should be P-384 (or when we switch to TLS 1.3, use Curve25519)

Reply:
Yeah, this is hard-coded to P-384 elsewhere.

Reply:
Yes - This is a dummy at the moment.